e2e/tests/workspace/workspace-actions-menu.spec.ts

```diff
@@ -0,0 +1,89 @@
+import WorkspaceDataPage from 'app/page/workspace-data-page';
+import {WorkspaceAccessLevel} from 'app/text-labels';
+import {findOrCreateWorkspace, signInWithAccessToken} from 'utils/test-utils';
+import WorkspaceAboutPage from 'app/page/workspace-about-page';
+import {makeWorkspaceName} from 'utils/str-utils';
+import {config} from 'resources/workbench-config';
+import {waitWhileLoading} from 'utils/waits-utils';
+import {logger} from 'libs/logger';
+
+describe('Workspace Share Modal', () => {
```

Comment:
Tests taken from deleted `workspace-actions-share.spec` and refined.

Reply:
The name of this new file does not make sense.  How about workspace-share-modal or similar?

Reply:
right, they're not matching. Updating test file name to `workspace-share-modal.spec`.